controllers/comments.controller.js

```diff
@@ -24,4 +28,14 @@ exports.postComment = (req, res, next) => {
       res.status(201).send({ comment });
     })
     .catch(next);
-    }
+};
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
another method you can use when sending nothing back, is `.sendStatus(204)`, this will set the status and send it with nothing all in one 👍